docs/_source/tutorials-examples/population-synthesis/bbh_analysis.ipynb

```diff
@@ -289,29 +250,79 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Transient population\n",
+    "### Underlying mass\n",
     "\n",
-    "Although we now have selected all binaries with a BBH merger, we don't have the extact moment of the merger yet.\n",
+    "Before we move on, we have to discuss the occurrence rate of each binary in the population.\n",
     "\n",
-    "For this we will create a `TransientPopulation`.\n",
-    "This class is used to hold information about a specific event/moment in time.\n",
+    "We have sampled several distributions at the start of our simulations.\n",
+    "These include:\n",
     "\n",
-    "In our case, this is the moment of \"CO_contact\". However, we might want to store \n",
-    "and calculate some additional values, such as the $M_\\mathrm{chirp}$ or $\\chi_\\mathrm{eff}$.\n",
+    "- Initial mass function\n",
+    "- Period\n",
+    "- mass ratio\n",
+    "- binary fraction\n",
```

Comment:
Could you add a warning here, that the non sampled parts of the period and mass ratio/secondary masses are currently not taken into account?

Reply:
Changed the warning right below to also include this